src/rpdk/core/fragment/generator.py

```diff
@@ -105,7 +105,7 @@ def __get_cfn_lint_matches(raw_fragment):
         filename = "temporary_fragment.json"
 
         with open(filename, "w") as outfile:
-            json.dump(raw_fragment, outfile, indent=4)
+            json.dump(raw_fragment, outfile, indent=4, default=str)
```

Comment:
does this affect any other values in templates? i guess cloudformation interprets everything as a string so it shouldnt matter?

Reply:
I'm not aware of another case, but I think exporting as string is the right behaviour. This is just about getting the linting results when creating a module, so the blast radius of this change is pretty small.